stable/cert-exporter/README.md

```diff
@@ -8,94 +8,169 @@ This chart deploys a DaemonSet to export the metrics of cert expiration on hosts
 
 ## Installing the Chart
 
-To install the chart with the release name `my-release`:
+### Using Helm 2
+
+To install the chart with the release name <RELEASE_NAME>:
+
+```bash
+$ helm install \
+      --name <RELEASE_NAME> \
+      suse/cert-exporter
+```
+
+### Using Helm 3
 
 ```bash
-$ helm install --name my-release suse/cert-exporter
+$ helm install <RELEASE_NAME> \
```

Comment:
```suggestion
$ helm3 install <RELEASE_NAME> \
```

Reply:
All the doc use the command `helm` either helm2 or helm3
https://github.com/SUSE/doc-caasp/pull/896/files